src/components/profile/hardware_devices/HardwareDevices.js

```diff
@@ -70,7 +70,7 @@ const HardwareDevices = ({ recoveryMethods }) => {
     const publicKeys = keys.map(key => key.public_key)
     const hasOtherMethods = publicKeys.some(key => recoveryKeys.includes(key))
     const hasLedger = userRecoveryMethods.filter(method => method.kind === 'ledger').map(key => key.publicKey).some(key => publicKeys.includes(key))
-    const ledgerIsConnected = account.ledgerKey !== null && hasLedger;
```

Comment:
@Patrick1904 why is this change needed, is info about `ledger` not added correctly to recovery methods?

Reply:
@vgrichina since the account can be secured with Ledger outside of the NEAR Wallet, it's not a guarantee that contract helper as the record. So if the user logs in with Ledger in the NEAR Wallet, we need to show it as 'active' despite no record in contract helper. 

In general it seems to make more sense too to show Ledger as enabled/connected if there's a Ledger key present, since that's the ultimate truth.